docs/_templates/module_sets.tmpl

```diff
@@ -0,0 +1,21 @@
+.. -*- mode: rst -*-
+
+
+
+{% if data['module_set_constraints']%}
+.. list-table:: Modules
+   :header-rows: 1
+
+   * - .. centered:: Module
+     - .. centered:: Version constraint
+
+   {% for key, value in data["module_set_constraints"] | dictsort %}
+   * - .. centered:: {{ key }}
+     - .. centered:: {{ value }}
```

Comment:
How does this render if there is no version constraint?

Reply:
The template engine can handle it well it seems:
```json
...
"module_set_constraints": {
        "lsm": "~=2.27",
        "std": null
    }
...
```
renders as: 
![image](https://github.com/inmanta/inmanta-core/assets/98876549/2da8b784-f7a0-4d86-87ff-0238ddfcd7aa)